docs/docs/tutorial.md

```diff
@@ -220,6 +221,7 @@ First, add the third-party library **marked** to your application. This is a Jav
 ```html{7}
```

Comment:
The 7 should become an 8. We use the `{7}` to highlight a line in the generated markup (see it in action @ http://facebook.github.io/react/docs/tutorial.html#adding-markdown)